File: Baekjoon/python/10757.py
# math

"""
python3에서는 숫자 정수형이 integer뿐이고, integer의 경우 overflow를 고려하지 않아도 됨.
"""

"""
두 번째 풀이
"""
# input
print(sum(map(int, input().split())))

"""
첫 번째 풀이
"""
# input
a, b = map(int, input().split())

print(a + b)

File: Baekjoon/nodejs/10757.js
// math

const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin
});

rl.on("line", (line) => {
    const input = line.split(' ').map(num => BigInt(num));

    console.log((input[0] + input[1]).toString());

    /*
    입력 받을 때, BigInt형으로 받지 않고 string으로 받아 후에 BigInt로 변환 후 더해서 결과를 출력했을 때, 통과되지 않음.
    두 경우의 차이점은 아직 명확히 모르겠음.
    숫자 관련 문제는 고통받지 말고 python으로 해결하자.

    const input = line;

    console.log((BigInt(input[0]) + BigInt(input[1])).toString());
    */

    rl.close();
}).on("close", () => {
    process.exit();
});
